### PR TITLE
Add support for disabling modules via module --disable (#1655523)

### PR DESCRIPTION
**Pull request by M4rtinK**
Make it possible to disable modules via the module command --disable option.

Resolves: rhbz#1655523
Related: rhbz#1719347

### CODE REVIEW

**Comment by M4rtinK** [comment]
updated:
- bump pykickstart version in spec file to expected value, might need to be checked later if it matches actual pykickstart version
- fixed some pylint issues, remaining test errors seem to be caused by the pykickstart part of this change missing from test environment

File: pyanaconda/payload/dnfpayload.py [hunk 1]
```diff
@@ -453,23 +453,37 @@ def addDisabledRepo(self, ksrepo):
         self._add_repo(ksrepo)
         super().addDisabledRepo(ksrepo)
 
-    def _enable_modules(self):
-        """Enable modules (if any)."""
+    def _handle_modules(self):
```
Comment (jkonecny12) [inline pyanaconda/payload/dnfpayload.py]: I don't like handle, could we switch this to something more descriptive?

File: pyanaconda/payload/dnfpayload.py [hunk 2]
```diff
@@ -453,23 +453,37 @@ def addDisabledRepo(self, ksrepo):
...
                 module_spec = module.name
-            module_specs.append(module_spec)
+
+            if module.enable:
+                module_specs_to_enable.append(module_spec)
+            else:
+                module_specs_to_disable.append(module_spec)
+
+        # forward the module specs to disable to DNF
+        log.debug("disabling modules: %s", module_specs_to_disable)
```
[the remark this answers is not in the document]
Comment (M4rtinK) [reply pyanaconda/payload/dnfpayload.py]: If empty `module_specs_to_disable` should be an empty list, so `[]`. So no quotes needed IMHO. :)

File: pyanaconda/payload/dnfpayload.py (hunk 1 above)
Comment (M4rtinK) [reply pyanaconda/payload/dnfpayload.py]: At the moment (pre-payload modularization) it processes the kickstart module command, so let's rename it that way.

**Comment by M4rtinK** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
Ported to master with https://github.com/rhinstaller/anaconda/pull/2084

**Pull request merged**